compiler/optimizer/CopyPropagation.cpp

```diff
@@ -728,6 +728,14 @@ int32_t TR_CopyPropagation::perform()
             replaceCopySymbolReferenceByOriginalIn(copySymbolReference, equivalentDefNode, useNode, defNode);
```

Comment:
It seems like `replaceCopySymbolReferenceByOriginalIn()` is capable of substituting symrefs in descendants of `useNode` rather than (or in addition to) `useNode` itself. I *think* it will be OK to treat it as though it's substituting only in `useNode` itself, because `useNode` is a use node from use-def info, which as far as I know makes it necessarily a direct load of an auto or parameter. @vijaysun-omr, do you know whether that's accurate? (If so, then I think `replaceCopySymbolReferenceByOriginalIn()` could be drastically simplified, but that's not a suggestion for this PR.)

Reply:
@jdmpapin I agree with your understanding. I believe some other language implementations had tried to copy propagate expressions other than direct loads in that routine, but this was never tried (nor considered seriously) in production for any runtime based on OMR.  So, it could be cleaned up.